package.json

```diff
@@ -104,6 +103,6 @@
     "uint8arrays": "^1.1.0",
     "url": "^0.11.0",
     "validator": "^13.6.0",
-    "wrtc": "./wrtc-mock"
+    "wrtc": "^0.4.6"
```

Comment:
Just confirming that we don't use wrtc for anything. Is that correct? 

Is there some way to disable it and remove it as a dependency?

Reply:
we don't use wrtc for anything
sadly it's pulled as dependency by `ipfs` package.

It appeared in package.json of waggle, because this package causes big headache on Android, and Wiktor needs to enforce a local "mocked" version (which is basically empty). Sadly this broke build of zbay, so Emilia reverted this override here and Wiktor moved this manipulation to "build script" that he is running in Termux.